### ISSUE TITLE
stands appear to be affected by effects that pushes items around such as fans, wind,  etc

### ISSUE THREAD

**Issue by ghost** [ganymedes01/Et-Futurum#51]
not sure what's causing this but armor stands have a tendency to walk away 1 block or so every time i travel between dimensions  OR armors vanish off the stand. I think it may be related to a forge bug and maybe will not happen anymore.  please  keep this open for a bit  until i figure out why or how or if it's fixed trough a forge update.  because i noticed other odd similar entity behaviours like bibliocraft shelves kept flipping on me. right now is a big exclamation mark

**Comment by ghost** [comment]
yes CONFIRMED. is still happening

seems empty armor stands try to run away from you

see the attached pic.  

the two stands were originally placed by me by the wall beside the chest you can see a bit on the right the corner. right beside that wall that's where the stands used to be. then i left and when i came back they walked away. notice how they are clipping in each other . seems to me like the stands are afraid of you and are running away. maybe they have some living entity in them inherited i am not sure how they are coded. 
![screen shot 2015-11-24 at 8 31 35 am](https://cloud.githubusercontent.com/assets/6987027/11368126/e5ae1cfa-9285-11e5-84a8-f222d601ed70.png)

**Comment by ghost** [comment]
OK. FIGURED OUT the issue.
turns out they are affected by effects that move entities around. for example say knockback or wind will knock them back.

can you make the stands un-moveable in the code so they are not affected by anything once placed stay there forever?

one of my mods has an effect  to push items around which incidentally affects stands

**Comment by buttilda** [comment]
If I did that then they wouldn't behave like the 1.8 ones. They must be movable otherwise pistons, slime blocks, gravity, and etc won't affact them.

**Comment by ghost** [comment]
hmm. i can't use them since i got so many blocks in my game that push entities around and items suction things and so forth all my stands keep blowing away.

**Comment by ghost** [comment]
@ganymedes01  could you make it so you can lock them in or make a version of them that is not affected by gravity say add a cobblestone block to them to make the immovable

**Comment by buttilda** [comment]
Only if they exist in vanilla
The point of Et Futurum is only to backport features, I don't change them or tweak them (unless necessary due to minecraft's 1.7 limitations).

Bibliocraft adds armour stands that aren't movable, use theirs instead :)

**Comment by Entoarox** [comment]
1.8 armor stands have the `NoGravity` boolean tag that makes it so they are not affected by gravity in any way.

**Comment by buttilda** [comment]
Et Futurum armourstands also have that tag

**Comment by Entoarox** [comment]
Last I tried it the stand ignored it... Hold on, let me try again then...

Edit: You are indeed correct, Must have mistyped the data tag in the past then...
